cookbooks/bach_opentsdb/templates/default/etc_initd_opentsdb.erb

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh -e
+
+###############################################
+##   WARNING - THIS FILE IS MANAGED BY CHEF  ##
+## DO NOT MODIFY BY HAND, OR YOU'LL BE SORRY ##
+###############################################
+
+#
+# Modified from original source: Elastic Search
+# https://github.com/elasticsearch/elasticsearch
+# Thank you to the Elastic Search authors
+#
+### BEGIN INIT INFO
+# Provides:          opentsdb
+# Required-Start:    $network $named
+# Required-Stop:     $network $named
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Starts OpenTSDB TSD
+# Description:       Starts an OpenTSDB time series daemon
+### END INIT INFO
+
+PATH=/bin:/usr/bin:/sbin:/usr/sbin
+NAME=opentsdb
+TSD_USER=opentsdb
+TSD_GROUP=opentsdb
+DAEMON=/usr/share/opentsdb/bin/tsdb
+DAEMON_OPTS=tsd
+PID_FILE=/var/run/$NAME.pid
+
+# Maximum number of open files
+MAX_OPEN_FILES=65535
+
+. /lib/lsb/init-functions
+
+# The first existing directory is used for JAVA_HOME
+# (if JAVA_HOME is not defined in $DEFAULT)
+JDK_DIRS="/usr/lib/jvm/java-7-oracle /usr/lib/jvm/java-7-openjdk \
```

Comment:
Qhy not use a Java home attribute? (And we can pass in the one drom BACH Hadoop in the wrapper)?

Reply:
This is just the default init script. It's untouched, because a few lines below is where `/etc/default/opentsdb` gets sourced and this gets overridden anyway (see line 18 of the `etc_default_opentsdb.erb` template).

Reply:
I'm wondering if it's best to no touch the ones provided by the debian package at all even though they're the same bits